src/ui/hotkey_overlay.rs

```diff
@@ -125,7 +125,12 @@ impl HotkeyOverlay {
     }
 }
 
-fn format_bind(binds: &[Bind], mod_key: ModKey, action: &Action) -> Option<(String, String)> {
+fn format_bind(
+    binds: &[Bind],
+    mod_key: ModKey,
+    action: &Action,
+    hide_unbound: bool,
```

Comment:
I feel like it'll be better to filter out unbound keys in render() before calling this function, instead of introducing this extra argument.

Reply:
i agree. i'll have a look into this